lib/mastodon/media_cli.rb

```diff
@@ -113,5 +113,24 @@ def usage
       say("Imports:\t#{number_to_human_size(Import.sum(:data_file_size))}")
       say("Settings:\t#{number_to_human_size(SiteUpload.sum(:file_file_size))}")
     end
+
+    desc 'lookup', 'Lookup a toot by passing a media URL'
+    def lookup
+      prompt = TTY::Prompt.new
+
+      exit(1) unless (url = prompt.ask('Please enter a URL to the media to lookup:', required: true))
+
+      id = url
+           .split('/')[0..-2]
+           .select do |key|
+             key.scan(/\D/).empty?
+           end
+           .join('')
+
+      attachment = MediaAttachment.find(id)
+
+      prompt.say('The corresponding media object you referenced could not be found, perhaps the toot was deleted?') unless attachment
```

Comment:
these two lines should be an if/else statement, there's no reason to display the url if the attachment doesn't exist

Reply:
Will update things.